lib/train/extras/command_wrapper.rb

```diff
@@ -29,6 +29,8 @@ class LinuxCommand < CommandWrapperBase
     Train::Options.attach(self)
 
     option :sudo, default: false
+    option :shell, default: false
```

Comment:
Its a little bit sub-optimal to switch types of a variable, I would prefer `nil` here, which would lead to the default